torchtext/data/functional.py

```diff
@@ -114,3 +115,72 @@ def _internal_func(txt_iter):
         for line in txt_iter:
             yield sp_model.EncodeAsPieces(line)
     return _internal_func
+
+
+def compute_ngrams(tokens, n):
```

Comment:
There is `ngram_iterator` [here](https://github.com/pytorch/text/blob/1f2bd140a700311b3c0d9f8e4a832953896981cd/torchtext/data/utils.py#L204)